apps/site/assets/ts/projects/components/RoutePill.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactElement } from "react";
```

Comment:
I think we might already have a component that does this somewhere... although I maybe it's similar but not the same. See `apps/site/assets/ts/stop/components/RoutePillList.tsx`

Reply:
@ryan-mahoney yeah, this `RoutePillList` is similar in intent to the other but implemented somewhat differently; its appearance is also somewhat different.